decidim-admin/app/views/decidim/admin/organization/_form.html.erb

```diff
@@ -74,6 +74,28 @@
 
 <div class="row">
   <div class="columns xlarge-6">
+    <%= form.file_field :official_img_header %>
+    <% if current_organization.official_img_header.present? %>
+      <label><%= t('.current_image') %></label>
+      <%= image_tag current_organization.official_img_header.url %>
```

Comment:
Careful with this, as it will span over the whole column and break the format. Please use one of the thumbnails instead.